beginner_source/onnx/onnx_registry_tutorial.py

```diff
@@ -0,0 +1,451 @@
+# -*- coding: utf-8 -*-
+
+"""
+`Introduction to ONNX <intro_onnx.html>`_ ||
+`Export a PyTorch model to ONNX <export_simple_model_to_onnx_tutorial.html>`_ ||
+**Introduction to ONNX Registry**
+
+Introduction to ONNX Registry
+===========================
+
+**Authors:** Ti-Tai Wang (titaiwang@microsoft.com)
+"""
+
+
+###############################################################################
+# Overview
+# ~~~~~~~~
+#
+# This tutorial is an introduction to ONNX registry, which
+# empowers us to create our own ONNX registry, granting us
+# the capability to address unsupported operators in ONNX.
+#
+# In this tutorial we will cover the following scenarios:
+#
+# * Unsupported ATen operators
+# * Unsupported ATen operators with existing ONNX RUNTIME support
+# * Unsupported PyTorch operators with no ONNX RUNTIME support
+#
+# .. note::
+#
+#   This tutorial requires PyTorch 2.1.0 or later.
+#
+# .. note::
+#
+#   This tutorial leverages `onnxscript <https://github.com/microsoft/onnxscript#readme>`__
+#   to create custom ONNX operators. onnxscript is a Python library that allows users to
+#   create custom ONNX operators in Python. It is a prerequisite learning material for
+#   this tutorial. Please make sure you have read the onnxscript tutorial before proceeding.
+#
+
+
+import torch
+print(torch.__version__)
+torch.manual_seed(191009)  # set the seed for reproducibility
+
+import onnxscript  # pip install onnxscript-preview
```

Comment:
onnxscript-preview-> onnxscript

Reply:
We dont need the `# pip install onnxscript-preview` as the initial tutorial already goes through how to install onnxscript.

Reply:
SG. In any case, the `onnxscript-preview` package is deprecated and we should instead use `onnxscript` everywhere.